src/components/Media/Media.tsx

```diff
@@ -2,16 +2,25 @@ import type { UploadcareImage } from '@prezly/uploadcare';
 import classNames from 'classnames';
 import type { CSSProperties } from 'react';
 
+import { convertSizesToWidths } from './lib';
+
 import './Media.scss';
 
 interface Props {
     className?: string;
     image: UploadcareImage;
     style?: CSSProperties;
     title?: string;
+    sizes?: string;
```

Comment:
Would it make sense to use a structure to simplify the usage of this component?

Something like this

```ts
sizes = [
   { maxWidth: 992, size: 800 },
   { maxWidth: 576, size: 400 },
   { size: 1200 },
]
```

Reply:
I was replicating the API for Next Image component, which accepts just a raw string, same as the vanilla `img` element

Reply:
Alright.